src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -120,13 +120,6 @@ const WorkspaceTabs = ({
   ])
 }
 
-// v2 workspaces may have been migrated from v1 workspaces, in which case the googleProject
-// associated with runtimes on GCP will not match the workspace googleProject. These
-// should be hidden from the user.
-export const isV1Artifact = _.curry((workspace, { googleProject, cloudContext }) => {
```

Comment:
I have a runtime running in a v1 workspace, but due to the filtering commented on below, it's not being returned. And we no longer will be returning v1 workspaces from Rawls, so the workspace this container is for will always be a v2 workspace.